src/web/mjs/connectors/CxC.mjs

```diff
@@ -0,0 +1,254 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class CxC extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'cxc';
+        super.label = 'CxC - Content x Creator';
+        this.tags = [ 'webtoon', 'chinese' ];
+        this.url = 'https://cxc.today';
+        this.api = 'https://api.cxc.today';
+        this.booksPerPage = 50;
+        this.uuid = null;
+        this.auth = {
+            accessToken: null,
+            refreshToken: null,
+            expireTime: null
+        };
+        this.requestOptions.headers.set('accept', 'application/json, text/plain, */*');
```

Comment:
Despite using fetchJSON, the accept header for requests made is `text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9`, which is not ideal for guaranteeing JSON when the server can provide it. I imagine this accept header works for more scraper-like connectors, but since this is almost exclusively interacting with a JSON API it shouldn't be necessary.

Since [`fetch`](https://github.com/manga-download/hakuneko/pull/6023/files#diff-74dacebfa1b84f4d1772f27f7308bb1e4830b46f348d9bb2487120792807f92dR136) is used for getting the images without using `this.requestOptions`, the accept header is `image/webp,image/apng,image/*,*/*` instead.

Reply:
Yeah you are right, my bad. That said, putting that accept header in all requests is a bit overkill when you have only 2 or 3 times in the code. 

Oh well, do what you want for this.